codespell_lib/data/dictionary_from_LO.txt

```diff
@@ -0,0 +1,3087 @@
+anevironment->environment
+animatior->animation
+animaton->animation
+aninate->animate
+anination->animation
+anisotrophically->anisotropically
+anitaliasing->antialiasing
+anitialising->antialiasing
+anlyzing->analyzing
+anologon->analogon
+anonymus->anonymous
+anormaly->abnormally
+anroid->android
+ansynchronous->asynchronous
+antaliasing->antialiasing
+anthor->another
+antialialised->antialialiased
+antialising->antialiasing
+an we->and we
+an will->and will
+an window->a window
+an with->and with
+anyhing->anything
+anynmore->anymore
+anythimng->anything
+aond->and
+aovid->avoid
+apapted->adapted
+aparently->apparently
+apeends->appends
+apllicatin->application
+apllication->application
+appedn->append
+appendent->appendend
+appendign->appending
+appens->appends
+appied->applied
+applay->apply
+appliable->applicable
+applicabel->applicable
+applicaten->application
+applicatin->application
+applicationevnet->applicationevent
+applikation->application
+applyable->applycable
+applycable->applicable
+applyied->applied
+applys->applies
+appon->upon
+apporpiate->appropriate
+apppend->append
+appplet->applet
+approbiate->appropriate
+approperiate->appropriate
+appropriatedly->appropriately
+approrpriate->appropriate
+approrpriately->appropriately
+appy->apply
+aproach->approach
+aprrovement->approvement
+aprtment->apartment
+aquires->acquires
+ararbic->arabic
+arays->arrays
+arbitraryly->arbitrary
+arbitraty->arbitrary
+arbritary->arbitrary
+archor->anchor
+arcticle->article
+aren's->aren't
+argumant->argument
+argumants->arguments
+argumentents->arguments
+arithemetic->arithmetic
+arithmetics->arithmetic
+arive->arrive
+aronud->around
+arragement->arrangement
+arrance->arrange
+arrary->array
+arrys->arrays
+artihmetic->arithmetic
+asceding->ascending
+aske->ask
+askes->asks
+asny->any
+asnychronoue->asynchronous
+aspected->expected
+asscoitaed->associated
+assiging->assigning
+assigmnent->assignment
+assing->assign
+assocations->associations
+associdated->associated
+associeted->associated
+assoiciative->associative
+asssuming->assuming
+assumtions->assumption
+asynchornous->asynchronous
+asynchrons->asynchronous
+asynchrounous->asynchronous
+asynchrounsly->asynchronously
+asynschron->asynchronous
+atemptively->attemptively
+atempts->attempts
+atevents->atevent
+atorecovery->autorecovery
+atribs->attribs
+atribut->attribut
+atrtributes->attributes
+attachen->attach
+attachted->attached
+attept->attempt
+attibuted->attributed
+attrbiute->attribute
+attribbute->attribute
+attribiute->attribute
+attribiutes->attributes
+attribtes->attribute
+attribues->attributes
+attribuite->attribute
+attribures->attributes
+attributei->attribute
+attributs->attributes
+attriubtes->attributes
+attrubute->attribute
+auccessive->successive
+auospacing->autospacing
+auot->auto
+auotmatic->automatic
+autmatically->automatically
+autocommiting->autocommitting
+autocoomit->autocommit
+autofilt->autofilter
+autofomat->autoformat
+autogrowwith->autogrowwidth
```

Comment:
not sure we should add this

Reply:
There are a lot of specific words just for LO.

That's the reason I didn't merge directly, but let other decide about what to exclude.